genesis.json

```diff
@@ -0,0 +1,7122 @@
+              "amount": "4225500000000"
+            }
+          ]
+        },
+        {
+          "address": "cosmos1l2scvs2wve6lfz2ujgvrlkrsnw6g68dxkcqg8y",
+          "coins": [
+            {
+              "denom": "usomm",
+              "amount": "2250000000000"
+            }
+          ]
+        },
+        {
+          "address": "cosmos1jv65s3grqf6v6jl3dp4t6c9t9rk99cd88lyufl",
+          "coins": [
+            {
+              "denom": "usomm",
+              "amount": "150000000000000"
+            }
+          ]
+        }
+      ],
+      "supply": [],
+      "denom_metadata": [     
+        {
+          "description": "The native staking token of the Sommelier Chain",
+          "denom_units": [
+            {
+              "denom": "usomm",
+              "exponent": 0,
+              "aliases": [
+                "microsomm"
+              ]
+            },
+            {
+              "denom": "msomm",
+              "exponent": 3,
+              "aliases": [
+                "millisomm"
+              ]
+            },
+            {
+              "denom": "somm",
+              "exponent": 6
+            }
+          ],
+          "base": "usomm",
+          "display": "somm"
+        }
+      ]
+    },
+    "capability": {
+      "index": "1",
+      "owners": []
+    },
+    "crisis": {
+      "constant_fee": {
+        "denom": "usomm",
+        "amount": "1000"
+      }
+    },
+    "distribution": {
+      "params": {
+        "community_tax": "0.000000000000000000",
+        "base_proposer_reward": "0.000000000000000000",
+        "bonus_proposer_reward": "0.000000000000000000",
+        "withdraw_addr_enabled": true
+      },
+      "fee_pool": {
+        "community_pool": [
+          {
+            "amount": "150000000000000.0",
+            "denom": "usomm"
+          }
+        ]
+      },
+      "delegator_withdraw_infos": [],
+      "previous_proposer": "",
+      "outstanding_rewards": [],
+      "validator_accumulated_commissions": [],
+      "validator_historical_rewards": [],
+      "validator_current_rewards": [],
+      "delegator_starting_infos": [],
+      "validator_slash_events": []
+    },
+    "evidence": {
+      "evidence": []
+    },
+    "genutil": {
+      "gen_txs": []
+    },
+    "gov": {
+      "starting_proposal_id": "1",
+      "deposits": [],
+      "votes": [],
+      "proposals": [],
+      "deposit_params": {
+        "min_deposit": [
+          {
+            "denom": "usomm",
+            "amount": "10000000"
+          }
+        ],
+        "max_deposit_period": "172800s"
+      },
+      "voting_params": {
+        "voting_period": "172800s"
+      },
+      "tally_params": {
+        "quorum": "0.005000000000000000",
+        "threshold": "0.500000000000000000",
+        "veto_threshold": "0.334000000000000000"
+      }
+    },
+    "gravity": {
+      "params": {
+        "gravity_id": "defaultgravityid",
+        "contract_source_hash": "",
```

Comment:
I feel like putting this in there would be a nice thing to do. I don't think its wired up anywhere, but 🤷‍♂️